charts/seed-monitoring/charts/grafana/templates/grafana-ingress.yaml

```diff
@@ -8,16 +8,11 @@ metadata:
     nginx.ingress.kubernetes.io/auth-realm: Authentication Required
     nginx.ingress.kubernetes.io/auth-secret: {{ .Values.ingress.authSecretName }}
     nginx.ingress.kubernetes.io/auth-type: basic
-{{- if eq .Values.role "users" }}
```

Comment:
As we now unify to single grafana, https://github.com/gardener/gardener/blob/54bdaaac6f6dd43cb034b367459cf95acfbf4ff9/pkg/operation/botanist/component/types.go#L47-L48 no longer make sense.

Can you also sync with @Kristian-ZH and @vlvasilev about the required logging adaptations?

For example https://github.com/gardener/gardener/blob/v1.57.1/docs/extensions/logging-and-monitoring.md#how-to-expose-logs-to-the-users no longer makes sense.